tests/introspection/ApiTypoTest.cs

```diff
@@ -323,6 +323,7 @@ protected ApiTypoTest ()
 			"Ntsc",
 			"nuint",
 			"Ndef",
+			"Noi", // From NoiOSAttribute
```

Comment:
hmm... that was not caught before because it was turned into a `[Unavailable]` right ?

Reply:
Correct. But I added [NoiOS] and friends to match the [Mac] and friends that derive from the right attribute (since we're having to change tooling, let's make all of the common shortcuts work).

Reply:
https://github.com/xamarin/xamarin-macios/pull/3096/files#diff-8f0ef92138e06cd064a011ea8aad8d8bR279